app/src/main/java/travelbook/android/app/data/MemoriesDao.java

```diff
@@ -9,7 +14,55 @@
  */
 public class MemoriesDao {
 
+    private final TravelBookDatabaseOpenHelper travelBookDatabaseOpenHelper;
+    String selection = "is_primary != ?";
+    String[] selectionArgs = {"null"};
+
+    public MemoriesDao(TravelBookDatabaseOpenHelper travelBookDatabaseOpenHelper) {
+        this.travelBookDatabaseOpenHelper = travelBookDatabaseOpenHelper;
+    }
+
     public List<Memory> fetchAll() {
-        return Collections.emptyList();
+        List<Memory> memoryList = new ArrayList<Memory>();
+
+        SQLiteDatabase db = travelBookDatabaseOpenHelper.getWritableDatabase();
```

Comment:
There is no intention to write something into the database.
Use .getReadableDatabase instead